@here/harp-map/lib/BuiltinMap.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * This class is an alias for standard Map class that exists in Node and Browser environment.
+ *
+ * Required because as for now thre is no way to reliably alias name from global module from
+ * scope that redefines this name (and Map.ts defines class Map).
+ *
+ * @see https://github.com/Microsoft/TypeScript/issues/983
+ * @see https://stackoverflow.com/questions/51319046
+ */
+export const BuiltinMap = Map;
```

Comment:
Why rewrite the existing `Map` object? How does the user know that he now has to call `BuiltinMap`? Why would we be the only ones to do it that way? Can't we easily find another name than `Map`?

Reply:
Use doesn't need to know anything about this atrocity. This was needed only in `Map.ts` to use real "standard" Map, it is leftover which AFAIRis not needed anymore.

Why `Map` ... because, everyone does this ?
* `mapboxgl.Map` - https://docs.mapbox.com/mapbox-gl-js/overview/
 `google.maps.Map` - https://developers.google.com/maps/documentation/javascript/tutorial
...

`Map` works well in pre-ES6 world, where all JS libraries were delivered in some namespace.
Like, our `harp` namepace from `harp.js` bundle.
In Es6 world, when user is expecte to
```
import { Map } from "whatever";
```
This becomes really ugly, because user has to either rename it to say "HarpMap", or import whole module as namespace. Both are suboptimal.

Reply:
Ok. About a solution: would it be possible to rename it to something else for the modular version, and export it `as Map` for the namespaced version?